plume-models/src/blogs.rs

```diff
@@ -185,43 +185,43 @@ impl Blog {
         CustomGroup::new(blog, ap_signature)
     }
 
-    pub fn update_boxes(&self, conn: &PgConnection) {
+    pub fn update_boxes(&self, conn: &Connection) {
         let instance = self.get_instance(conn);
         if self.outbox_url.len() == 0 {
             diesel::update(self)
                 .set(blogs::outbox_url.eq(instance.compute_box(BLOG_PREFIX, self.actor_id.clone(), "outbox")))
-                .get_result::<Blog>(conn).expect("Couldn't update outbox URL");
+                .execute(conn).expect("Couldn't update outbox URL");
```

Comment:
what's the difference between these methods?

Reply:
`get_result`(`s`) requires the DB backend to support `RETURNING` statements (so, only with Postgres). Too support both SQlite and Postgres we should emulate this function by first using `execute`, then loading the result in another query.